README.md

```diff
@@ -2,56 +2,168 @@
 
 ## Introduction
 
-**Pan-Canadian-Genome-Library/molecular-data-submission-workflow** is a bioinformatics pipeline that ...
+**Pan-Canadian-Genome-Library/molecular-data-submission-workflow** is a Nextflow pipeline that automates the validation, packaging, and submission of molecular genomics data and associated metadata to the Pan-Canadian Genome Library (PCGL) data repository. The pipeline ensures data integrity, validates metadata compliance, handles file uploads, and generates comprehensive submission receipts for tracking and audit purposes. The workflow has adopted [nf-core](https://nf-co.re/) framework and best practice guidelines to ensure reproducibility, portability and scalability.
 
-<!-- TODO nf-core:
-   Complete this sentence with a 2-3 sentence summary of what types of data the pipeline ingests, a brief overview of the
-   major pipeline sections and the types of output it produces. You're giving an overview to someone new
-   to nf-core here, in 15-20 seconds. For an example, see https://github.com/nf-core/rnaseq/blob/master/README.md#introduction
--->
+For detailed information about workflow components, prerequisites, and system architecture, see **[Complete Introduction Guide](docs/Introduction.md)**  with comprehensive workflow overview, subworkflows, modules and technical details.
 
-<!-- TODO nf-core: Include a figure that guides the user through the major workflow steps. Many nf-core
-     workflows use the "tube map" design for that. See https://nf-co.re/docs/contributing/design_guidelines#examples for examples.   -->
-<!-- TODO nf-core: Fill in short bullet-pointed list of the default steps in the pipeline -->
+## Pipeline Overview
+[TODO] - add a workflow diagram
 
-## Usage
+The workflow consists of five main stages:
+
+1. **Dependency Checking** - Validates input files, metadata completeness, and submission prerequisites
+2. **Metadata Payload Generation** - Creates standardized JSON payloads from input metadata
+3. **Data Validation** - Validates data files and metadata against PCGL schemas and requirements  
+4. **Data Upload** - Handles secure file transfer and metadata submission to PCGL repositories
+5. **Receipt Generation** - Creates detailed batch receipts and summary reports for submission tracking
+
+## Prerequisites
+
+### System Requirements
+
+- **Nextflow**: Version 22.04.0 or newer with DSL2 support
+- **Container Engine**: One of the following:
+  - Docker (recommended)
+  - Singularity/Apptainer 
+  - Conda (alternative, but containers preferred)
+- **Java**: Java 17 (or later, up to 24)
+- **Bash**: Bash 3.2 (or later)
+- **Memory**: Minimum 8GB RAM recommended
+- **Storage**: Sufficient disk space for input data, intermediate files, and outputs
+
+### Access Requirements
+
+- **PCGL Access Token**: Valid authentication token with submission permissions for your study
+- **Study Registration**: Your study must be registered in the PCGL system
```

Comment:
`Your study must be registered in the PCGL system. Contact PCGL Admin for more info.`